### DISCUSSION TITLE
Numbered list in reverse order

### DISCUSSION THREAD

**Discussion by Koushikphy** [mmistakes/minimal-mistakes#4904] [Q&A]
Is it possible to add a list or items that shows up as a reverse numbered list?

**Comment by ryan-p-randall** [comment]
I haven't been able to make reversed lists work when written in Markdown, but it is possible if you write the list in HTML.

In HTML, you need to add [`reversed`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol#reversed) to the opening `<ol>` tag, like in this example:
```html
<ol reversed>
    <li>Apples</li>
    <li>Bananas</li>
    <li>Coconuts</li>
    <li>Dragonfruit</li>
    <li>Eggplants</li>
</ol>
```
I tried using a few different ways of adding that `reversed` attribute using [Kramdown syntax for attributes](https://kramdown.gettalong.org/syntax.html#attribute-list-definitions), but couldn't get them to work. Here's an example of what I tried:
```markdown
1. Bananas
2. Coconuts
3. Dragonfruit
4. Eggplants
5. Apples
{:other: reversed="true"}
```

Here's how the above two lists are rendered on my site:
<img width="417" alt="two ordered lists, with the first successfully numbered from 5 to 1 and the second instead numbered from 1 to 5" src="https://github.com/user-attachments/assets/15636356-49fc-41c4-a2f0-9c784ed0c227">

When I use my browser inspector tools, the `reversed="true"` just disappears. My best guess is that Kramdown doesn't know how to use this kind of attribute for lists.

I hope that helps!
